test/framework/src/main/java/org/opensearch/test/OpenSearchIntegTestCase.java

```diff
@@ -787,6 +787,7 @@ protected Settings featureFlagSettings() {
         }
         // Enabling Telemetry setting by default
         featureSettings.put(FeatureFlags.TELEMETRY_SETTING.getKey(), true);
+        featureSettings.put(FeatureFlags.ZSTD_COMPRESSION_SETTING.getKey(), true);
```

Comment:
I don't think we should set this test wide. Let's keep it disabled and only re-enable in the explicit ZStd tests.